app/services/docusign/create_envelope_from_template.rb

```diff
@@ -0,0 +1,17 @@
+class Docusign::CreateEnvelopeFromTemplate < ApplicationService
+  DocusignError = Class.new(StandardError)
+
+  attr_reader :payload
+
+  def initialize(payload)
+    @payload = payload
+  end
+
+  def call
+    client = DocusignRest::Client.new
+    result = client.create_envelope_from_template(payload)
+    raise DocusignError, result['message'] if result['errorCode']
```

Comment:
If there is an error, will we need to log the errorCode in order to debug?

Reply:
errorCode doesn't seem useful in my opinion, the message should be enough:

examples:
`{"errorCode"=>"INVALID_GROUP_NAME", "message"=>"No group name was provided."}`
`{"errorCode"=>"SIGNING_GROUP_INVALID", "message"=>"Invalid signing group supplied."}`